fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseMaker.java

```diff
@@ -63,13 +66,15 @@ public HttpResponseMaker(ObjectMappersSuite objectMappers, DimensionDictionary d
      * @param preResponse  PreResponse object which contains result set, response context and headers
      * @param responseFormatType  The format in which the response should be returned to the user
      * @param uriInfo  UriInfo of the request
+     * @param apiRequest apiRequest needed for serialization writer
      *
      * @return Completely built response with headers and result set
      */
     public javax.ws.rs.core.Response buildResponse(
             PreResponse preResponse,
             ResponseFormatType responseFormatType,
-            UriInfo uriInfo
+            UriInfo uriInfo,
+            ApiRequest apiRequest
```

Comment:
Not so sure if `UriInfo` and `ResponseFormatType` still needed as we can get them from ApiRequest.